index.html

```diff
@@ -14,10 +14,18 @@
         </div>
         <a class="menuim" href="#"><img src="./img/menu.svg" alt="Menu"></a>
         <ul class="nav_ul">
-            <li>Portfolio</li>
-            <li>About</li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">   
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
```

Comment:
- [ ] Kindly set the margin to match the margins and padding provided by the Figma template for your X button and your hamburger.
Your design:
![image](https://user-images.githubusercontent.com/61919617/174821488-1a7a60bd-3898-430b-8d1d-0762feef8b84.png)

Figma design:
![image](https://user-images.githubusercontent.com/61919617/174821653-8ed84f0c-9a67-40fe-80c1-942e1014c706.png)